reporter/src/main/kotlin/reporters/OpossumReporter.kt

```diff
@@ -106,23 +106,17 @@ class OpossumReporter : Reporter {
                     "name" to source,
                     "documentConfidence" to 80
                 ),
-
```

Comment:
I cannot really tell whether the empty lines really were placed arbitrarily... maybe some grouping was intended by @maxhbr?

Reply:
I've dropped the term "arbitrary" from the commit message.